twitter_bot/Commands/kc.py

```diff
@@ -29,6 +29,9 @@ def kc(rsn):
             else:
                 manual_incorrect = r['manual']['incorrect_reports']
                 accuracy = " They have a report accuracy of " + str(round((manual_reports-manual_incorrect)/(manual_reports)*100, 2)) + "%"
+                
+            if reports == 0:
+                return
```

Comment:
we need to return a response pretty sure

Reply:
Edit: Response is validated, if Response is None nothing will be send
https://github.com/Bot-detector/Bot-detector-Twitter/blob/d5f10ed98e1b67718eacc9d7faec2dc9673d4438/twitter_bot/Tweet/Tweet.py#L62